modules/Template/PortalPublication.yaml

```diff
@@ -0,0 +1,70 @@
+#######################################
+# Portal          
+#######################################
+
+# Note: this template definition uses the alternate syntax with 'attribute'
+
+classes:
+  PublicationTemplate:
+    is_a: Template
+    description: >
+      A generalized data model template for Publications to be hopefully used across dccs
+    notes:
+    - This structure is based on a CSV file from MC2 annotations with customized fields as attributes.
+    attributes:
+      grantNumber:
+        range: string
+        required: true
+      consortiumName:
```

Comment:
Remove `consortiumName` as it shouldn't be the base class -- not everyone will use it; we certainly don't